### DISCUSSION TITLE
[Feature Request]: 分类策略配置文件，可以支持两个相同条件的类型配置吗

### DISCUSSION THREAD

**Discussion by leyung** [jxxghp/MoviePilot#3472] [讨论]
### 当前程序版本

v1.9.1-1

### 运行环境

Docker

### 功能改进类型

主程序

### 功能改进

我想要分类策略能够区分儿童电视剧和儿童动漫，我设想的分类配置如下：

```
tv:
  // 两个genre_ids分开，达到并且的效果
  儿童动漫:
    genre_ids: '10762'
    genre_ids: '16'
  儿童电视剧:
    genre_ids: '10762'
``` 

儿童电视剧例如：[家有儿女](https://www.themoviedb.org/tv/40521-home-with-kids)
儿童动漫例如：[阿凡提的故事](https://www.themoviedb.org/tv/69195)

**Comment by cng10001** [comment]
感觉可以把，你拿这两部添加订阅测试下呢。

**Comment by leyung** [comment]
> 感觉可以把，你拿这两部添加订阅测试下呢。

不行，我添加了两个genre_ids，媒体类别就只有全部了

**Comment by cng10001** [comment]
媒体类别只有全部跟添加两个genre ids没关系吧
![image](https://github.com/jxxghp/MoviePilot/assets/115156545/abcc7124-22e2-4fc0-bd4f-e6f363dc8d8a)
它默认二级分类上面不就是有多个genre ids吗

**Comment by leyung** [comment]
> 媒体类别只有全部跟添加两个genre ids没关系吧 ![image](https://private-user-images.githubusercontent.com/115156545/333843775-abcc7124-22e2-4fc0-bd4f-e6f363dc8d8a.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTY2OTA5NjksIm5iZiI6MTcxNjY5MDY2OSwicGF0aCI6Ii8xMTUxNTY1NDUvMzMzODQzNzc1LWFiY2M3MTI0LTIyZTItNGZjMC1iZDRmLWU2ZjM2M2RjOGQ4YS5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwNTI2JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDUyNlQwMjMxMDlaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT0xYmEzMTFkZjkzNDY5MjUzYjIzZmU0NmQzMTBlYzYyMTBlNDVkNjY5NGUyOTQxMmEzZmY5M2E5NWNmMjYxNzM1JlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.38-yAyV6IYUgfytEaZsZ-cCpWEPYH1kXzHH2lhwVC2U) 它默认二级分类上面不就是有多个genre ids吗

你看一下我发点例子，是在一个分类下同时写两行genre ids
![image](https://github.com/jxxghp/MoviePilot/assets/38756238/a9e8f679-8fee-47bf-b05f-385838187bc7)

**Comment by torashen** [comment]
儿童动漫:
genre ids: "10762,16"

我理解“或”和"和”关系的应该这么写。。

**Comment by leyung** [comment]
> 儿童动漫: genre ids: "10762,16"
> 
> 我理解“或”和"和”关系的应该这么写。。

这是“或”的关系，“和”的关系目前同一个genre_ids指标是无法实现的
